### PR TITLE
Category Screen Done Except for search icon and the action of every c…

### PR DESCRIPTION
**Pull request by Ali-Elsadany**
need to know what the category will navigate to ?

### CODE REVIEW

File: lib/core/utils/app_strings.dart [hunk 1]
```diff
@@ -46,6 +46,12 @@ class AppStrings {
   static const String bookaDoctor = "bookaDoctor";
   static const String bookaDate = "bookaDate";
   static const String selectaTime = "selectaTime";
+  static const String speech = "speech";
+  static const String autism = "autism";
+  static const String learningDifficulties = "learningDifficulties";
+  static const String intelligenceTest = "intelligenceTest";
+  static const String behaviorModification = "behaviorModification";
+  static const String skillsDevelopment = "skillsDevelopment";
```
Comment (SA33ED) [inline lib/core/utils/app_strings.dart]: This Data Will Be In API

File: lib/core/utils/app_assets.dart [hunk 2]
```diff
@@ -8,4 +8,10 @@ class AppAssets{
   static const String onBoard2 = '${basePath}onboard2.png';
   static const String onBoard3 = '${basePath}onboard3.png';
   static const String check = '${basePath}check.png';
+  static const String aut = '${basePath}autism.png';
+  static const String behMod = '${basePath}behavior modification.png';
+  static const String intTest = '${basePath}intelligence test.png';
+  static const String learnDiff = '${basePath}learning difficulties.png';
+  static const String skillsDevelop = '${basePath}skills development.png';
+  static const String speech = '${basePath}speech.png';
```
Comment (SA33ED) [inline lib/core/utils/app_assets.dart:16]: This Images Will Be In API

File: lib/features/category/screen/category_screen.dart [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+}
```
Comment (SA33ED) [inline lib/features/category/screen/category_screen.dart]: Add Screen Utils , Make Your UI Responsive

**Review by SA33ED — COMMENTED** [review]
مرفوض

**Comment by Ali-Elsadany** [comment]
in the edited commit ,i just removed the strings from ar , en and app strings files and made the app more responsive but leaved images to test on it and when api is done i will change it and remove the assets

**Pull request merged**